rholang/src/test/scala/coop/rchain/rholang/interpreter/util/codec/Base58Spec.scala

```diff
@@ -0,0 +1,34 @@
+package coop.rchain.rholang.interpreter.util.codec
+
+import org.scalatest._
+import org.scalatest.prop._
+
+
+class Base58Spec extends FlatSpec with TableDrivenPropertyChecks with Matchers {
+  "encode empty array" should "be empty" in {
+    val input = Array[Byte]()
+    Base58.encode(input) should equal("")
+  }
+
+  val examples =
+    Table(
+      ("plain", "encoded"),
+      (0, "1"),
+      (58, "21"),
+      (58*58, "211"),
+      (58*58*58, "2111")
```

Comment:
I only tested a few simple cases with unit tests because I wrote generative tests that produce random Base58 encodings.

On the other hand you have a valid point that we should test for order or digits in the Base58 alphabet. I'll write a test for that

Reply:
I added an extra test including all the base58 digits